data/processSynapseJSON.ts

```diff
@@ -51,10 +52,26 @@ import CELLXGENE_MAPPINGS from './cellxgene-mappings.json';
 import UCSCXENA_MAPPINGS from './ucscxena-mappings.json';
 import ISBCGC_MAPPINGS from './isbcgc-mappings.json';
 import CUSTOM_MINERVA_STORY_MAPPINGS from './minerva-story-mappings.json';
-import AUTOMINERVA_ASSETS from './htan-imaging-assets.json';
+// import AUTOMINERVA_ASSETS from './htan-imaging-assets.json';
 import IDC_IMAGING_ASSETS from './idc-imaging-assets.json';
 import CDS_ASSETS from './cds_drs_mapping.json';
 
+const AUTOMINERVA_ASSETS_URL = 'https://d3p249wtgzkn5u.cloudfront.net/final-output/htan-imaging-assets-latest.json'
+
+async function fetchAutoMinervaAssets(): Promise<AutoMinerva[]> {
+    const response = await fetch(AUTOMINERVA_ASSETS_URL);
+    if (!response.ok) {
+        throw new Error(`Failed to fetch AUTOMINERVA_ASSETS: ${response.statusText}`);
+    }
+    return response.json();
+}
+
+const AUTOMINERVA_ASSETS = await fetchAutoMinervaAssets();
```

Comment:
This top-level await is causing a compile error for me:

```
TS1378: Top-level 'await' expressions are only allowed when the 'module' option is set to 'es2022', 'esnext', 'system', 'node16', or 'nodenext', and the 'target' option is set to 'es2017' or higher.
```

We should probably move this inside another function. I'll try to fix it.